src/types/index.ts

```diff
@@ -243,3 +243,160 @@ export interface ERC20 extends Contract {
     ): TransactionObject<boolean, [string, string, string | number | BN]>
   }
 }
+
+export interface StablecoinConverter extends Contract {
+  clone(): StablecoinConverter
+
+  methods: EpochTokenLocker['methods'] & {
+    getSecondsRemainingInBatch(): TransactionObject<string>
+
+    feeDenominator(): TransactionObject<string>
+
+    getPendingWithdrawAmount(user: string, token: string): TransactionObject<string, [string, string]>
+
+    requestWithdraw(
+      token: string,
+      amount: number | string | BN,
+    ): TransactionObject<void, [string, number | string | BN]>
+
+    getPendingDepositAmount(user: string, token: string): TransactionObject<string, [string, string]>
+
+    deposit(token: string, amount: number | string | BN): TransactionObject<void, [string, number | string | BN]>
+
+    getPendingWithdrawBatchNumber(user: string, token: string): TransactionObject<string, [string, string]>
+
+    TOKEN_ADDITION_FEE_IN_OWL(): TransactionObject<string>
+
+    feeToken(): TransactionObject<string>
+
+    currentPrices(arg0: number | string | BN): TransactionObject<string, [number | string | BN]>
+
+    orders(
+      arg0: string,
+      arg1: number | string | BN,
+    ): TransactionObject<
+      {
+        buyToken: BN
+        sellToken: BN
+        validFrom: BN
+        validUntil: BN
+        isSellOrder: boolean
+        priceNumerator: BN
+        priceDenominator: BN
+        usedAmount: BN
+        0: BN
+        1: BN
+        2: BN
+        3: BN
+        4: BN
+        5: BN
+        6: BN
+      },
+      [string, number | string | BN]
+    >
+
+    numTokens(): TransactionObject<string>
+
+    lastCreditBatchId(arg0: string, arg1: string): TransactionObject<string>
+
+    latestSolution(): TransactionObject<{
+      batchId: string
+      solutionSubmitter: string
+      feeReward: string
+      objectiveValue: string
+      0: string
+      1: string
+      2: string
+      3: string
+    }>
+
+    getBalance(user: string, token: string): TransactionObject<string, [string, string]>
+
+    getCurrentBatchId(): TransactionObject<string>
+
+    requestFutureWithdraw(
+      token: string,
+      amount: number | string | BN,
+      batchId: number | string | BN,
+    ): TransactionObject<void, [string, number | string | BN, number | string | BN]>
```

Comment:
Shouldn't this be in the EpochTokenLocker contract definition?

Reply:
you are right, typechain confused me with which methods are where
I'll sort them out in a separate PR as it doesn't break anything here